.openshift-ci/get_latest_release_versions.py

```diff
@@ -18,16 +18,17 @@ def filter_tags(rawtags):
 def make_image_tag():
     return subprocess.check_output(["make", "--quiet", "--no-print-directory", "tag"]).decode(encoding="utf-8")
 
-def extract_y_from_main_image_tag(mainimagetag):
-    return int(re.search(r"\d+\.(\d+)", mainimagetag).group(1))
+def extract_x_y_from_main_image_tag(mainimagetag):
+    x_y = re.search(r"(\d+)\.(\d+)", mainimagetag)
+    return int(x_y.group(1)), int(x_y.group(2))
 
 def get_latest_tags(tags, num_versions):
```

Comment:
After looking at this, I think that the function can be split in two: the first one finds the latest patch for each `major.minor`. The second function will query `make tag` and take the last `N` versions just before the tag.

Reply:
Done